### ISSUE TITLE
Test suits containing skipped tests are reported as failed

### ISSUE THREAD

**Issue by amakhrov** [litixsoft/karma-mocha-reporter#42]
If some other test fails, the Reporter lists the name of the skipped test suite in addition to the actually failed test.

<img width="398" alt="skipped-test-as-failed" src="https://cloud.githubusercontent.com/assets/1004015/13365760/c6241fd8-dc8b-11e5-9d64-99904d7e8be4.png">

The code of the test suite that is mistakenly reported here:
````
describe('"describe" containing a skipped "it"', function () {
	var spy;

	xit('test 1 - SKIPPED', function () {
		expect(spy).to.have.been.calledOnce();
	});

	it('test 2', function () {
	});
});
````

**Comment by 4kochi** [comment]
I was able to reproduce the problem. Can you please share a snippet which also contains the failing test. Thanks.

**Comment by amakhrov** [comment]
````
'use strict';

describe('Test Suit 1 (with a skipped test)', function () {
	xit('should fail', function () {
		assert.isTrue(false);
	});

	it('should pass', function () {
		assert.isTrue(true);
	});
});


describe('Test Suite 2', function () {
	it('should fail', function () {
		assert.isTrue(false);
	});
});
````

Output:
<img width="340" alt="skipped-test-as-failed" src="https://cloud.githubusercontent.com/assets/1004015/13374301/51409e6c-dd35-11e5-9ef6-90612bb66305.png">

Looks like this only happens when in Test Case 1 the skipped (`xit`) case is not the last one.

**Comment by 4kochi** [comment]
It works for me. 

<img width="570" alt="screen" src="https://cloud.githubusercontent.com/assets/1742426/13374943/3026c752-dd93-11e5-82c4-b8cc5c20d9eb.png">


But i am using jasmine for the tests. Which test framework and assertion library do you use? And do you have any of the options of the reporter enabled?

**Comment by amakhrov** [comment]
Mocha + Chai
Sorry, I didn't specify form the beginning - I assumed this is a mocha-specific repo (karma-**mocha**-report).

Full Karma config:
````
config.set({
	basePath: './media/source/scripts',
	frameworks: [ 'browserify', 'mocha', 'chai' ],
	files: ['**/*.test.js'],
	preprocessors: {
		'**/*.test.js': [ 'browserify' ]
	},

	browsers: ['PhantomJS'],

	browserify: {
		debug: true,
		transform: [ 'jadeify', 'envify' ]
	},

	reporters: ['mocha'],
	mochaReporter: {
		output: 'minimal'
	}

});
````

**Comment by 4kochi** [comment]
Ok, thanks. Now I can reproduce it. So it must be a mocha or chai specific problem. Since I always use jasmine I did not encountered the problem yet.

Btw, the reporter uses the api provided by karma, so it is actually test framework agnostic. :smile:

**Comment by amakhrov** [comment]
Sounds good :)
Thanks for the quick response!

**Comment by 4kochi** [comment]
Released as v1.2.3

**Comment by amakhrov** [comment]
Wow, such a fast turnaround! Impressive )
